models/demos/falcon7b/tests/unit_tests/test_falcon_matmuls_and_bmms_with_mixed_precision.py

```diff
@@ -59,7 +62,7 @@ def run_falcon_matmul_test(
             out_mem_config = ttnn.experimental.tensor.MemoryConfig(
                 ttnn.experimental.tensor.TensorMemoryLayout.INTERLEAVED, ttnn.experimental.tensor.BufferType.DRAM
             )
-    elif falcon_op == ttnn.experimental.tensor.falcon_dense_h_to_4h_matmul:
+    elif falcon_op == "falcon_dense_h_to_4h_matmul":
```

Comment:
its a little sad to see this being changed to a string

Reply:
I feel it is totally possible to use objects

- falcon_dense_4h_to_h_matmul
- falcon_dense_h_to_4h_matmul
- falcon_lm_head_matmul

Reply:
I used Enum